src/pages/NewTaskPage.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import withLocalize, {withLocalizePropTypes} from '../components/withLocalize';
+import compose from '../libs/compose';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import Navigation from '../libs/Navigation/Navigation';
+import ScreenWrapper from '../components/ScreenWrapper';
+import styles from '../styles/styles';
+import ONYXKEYS from '../ONYXKEYS';
+import * as ErrorUtils from '../libs/ErrorUtils';
+import Form from '../components/Form';
+import TextInput from '../components/TextInput';
+import Permissions from '../libs/Permissions';
+
+const propTypes = {
+    /** List of betas available to current user */
+    betas: PropTypes.arrayOf(PropTypes.string),
+
+    ...withLocalizePropTypes,
+};
+const defaultProps = {
+    betas: [],
+};
+
+const NewTaskPage = (props) => {
+    /**
```

Comment:
Wouldn't it be better to create a function component syntax instead of using the arrow method?

NAB

Reply:
I don't believe there's any difference in this use case - we seem to have both and so it's a matter of preference